packages/callout-block/src/utils/hasRichTextValue.ts

```diff
@@ -0,0 +1,21 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export const hasRichTextValue = (string?: string) => {
+    if (!string) {
+        return false;
+    }
+    const json = JSON.parse(string);
```

Comment:
this line will crash if the passed string is not a valid json (for example "`asdf`"). Can this happen or is it always a valid json?